package.json

```diff
@@ -85,12 +85,15 @@
     "prism-react-renderer": "1.3.5",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "ts-node": "10.8.2",
-    "typedoc": "0.23.5",
+    "ts-node": "10.9.1",
+    "typedoc": "0.23.10",
     "typescript": "4.7.4",
     "url-loader": "4.1.1"
   },
   "publishConfig": {
     "tag": "alpha"
+  },
+  "dependencies": {
+    "npm-check": "^6.0.1"
```

Comment:
Should this be a dev dependency or not included at all or is this intentional?

Reply:
@yaacovCR Good, catch.
I just forgot to add `--global` to a command.